packages/contracts/test/contracts/OVM/chain/OVM_CanonicalTransactionChain.spec.ts

```diff
@@ -1375,12 +1496,12 @@ describe('OVM_CanonicalTransactionChain', () => {
               const blockNumber =
                 (await getNextBlockNumber(ethers.provider)) - 50
 
-              contexts = [...Array(size)].map(() => {
+              contexts = [...Array(size)].map((el, idx) => {
```

Comment:
The small diffs in this test are necessary, because the test itself was actually generating non-increasing contexts, and was only passing because of the missing checks.